docs/DeveloperGuide.adoc

```diff
@@ -844,6 +897,137 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+[discrete]
+=== Use case: Add a to-do
+
+*MSS*
+
+1.  User requests to view tasks in a specific week
+2.  AddressBook shows tasks in the specified week
+3.  User requests to add a new task in a specific week
+4.  AddressBook adds a new task to the existing list
++
+Use case ends.
+
+*Extensions*
+
+* 1a. Specified week does not exist.
++
+[none]
+** 1a1. AddressBook shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Mark a to-do as complete
+
+*MSS*
+
+1.  User requests to view tasks in a specific week
+2.  AddressBook shows tasks in the specified week
+3.  User requests to complete a task in a specific week
+4.  AddressBook marks specified task as complete
++
+Use case ends.
+
+*Extensions*
+
+* 1a. Specified week does not exist.
++
+[none]
+** 1a1. AddressBook shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: find person
+
+*MSS*
+
+1.  User types find
+2.  ProgressCheck automatically shows the list dynamically without the user needing to press enter key
+3.  User need not need to type the whole name, substrings will generate results
+4.  ProgressCheck displays the necessary results
```

Comment:
This MSS is a bit weird, especially step 3. Recall what a MSS is and revise this MSS.